komga/src/main/kotlin/org/gotson/komga/domain/service/SeriesLifecycle.kt

```diff
@@ -240,15 +247,57 @@ class SeriesLifecycle(
       thumbnailsSeriesRepository.markSelected(thumbnail)
   }
 
+  fun markThumbnailAsSelected(series: Series, thumbnailId: String) {
+    thumbnailsSeriesRepository.findAllBySeriesId(series.id)
+      .forEach {
```

Comment:
Unnecessary as the repo method ensures all others are unselected when one is selected